live-examples/css-examples/basic-box-model/overscroll-behavior.css

```diff
@@ -15,7 +15,7 @@
 #example-element {
     width: 50%;
     height: 12em;
-    border: medium dotted #1b76c4;
+    border: medium dotted #009e5f;
```

Comment:
Sorry, I made a mistake in the list of contrast issues. This shouldn't be changed, because contrast looks good and also `overscroll-behavior-x` and `overscroll-behavior-y` use the same color.